DESCRIPTION

```diff
@@ -17,16 +17,16 @@ Imports:
     mlr3,
     mlr3misc,
     paradox,
-    R6
+    R6,
+    extraTrees
 Suggests:
     testthat,
-    rmarkdown,
-    randomForest
+    rmarkdown
```

Comment:
Is this really required? I guess this is just for building the documentation? But then it would also be necessary to include roxygen2 in Suggests?

Reply:
mlr-org/mlr3learnertemplate#1